charts/ocis/values.yaml

```diff
@@ -1305,76 +2782,178 @@ services:
     # -- Persistence settings.
     # @default -- see detailed persistence configuration options below
     persistence:
+      # @schema
+      # type: boolean
+      # required: true
+      # @schema
       # -- Enables persistence.
       # Needs to be enabled on production installations, except `features.externalUserManagement.enabled` equals `true`.
       # If not enabled, pod restarts will lead to data loss.
       enabled: false
+      # @schema
+      # type: boolean
+      # required: true
+      # @schema
       # -- Enables an initContainer to chown the volume.
       # The initContainer is run as root.
       # This is not needed if the driver applies the fsGroup from the securityContext.
       # The image specified in `initContainerImage` will be used for this container.
       chownInitContainer: false
+      # @schema
+      # type: [string, null]
+      # required: false
+      # @schema
       # -- Storage class to use.
       # Uses the default storage class if not set.
       storageClassName:
+      # @schema
+      # type: array
+      # items:
+      #   type: string
+      #   required: true
+      # @schema
```

Comment:
I think the `accessModes` are well-defined so we could use an enum here. Similar for the other persistence setting in services.

Reply:
fixed in https://github.com/owncloud/ocis-charts/pull/811/commits/a8569418f045054c0c7e30e444139d6d73d3449b